posthog/migrations/0184_delete_sessionsfilter.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.5 on 2021-11-29 14:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0183_groups_pg"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(name="SessionsFilter",),
```

Comment:
Not sure if we should back anything up / keep the model around. Would the filters have any future use?

Reply:
This is a good question. On cloud, this feature has very limited usage (excluding our team, 29 session filters have been made), so I feel pretty comfortable simply deleting it.

I guess the risk is that some self-hosted users make extensive use of saved session filters, and they want to downgrade after upgrading to get back to the old experience. I feel like this likelihood is pretty small - maybe we include a clear note in the release notes? @paolodamico

Reply:
Sounds like a plan @rcmarron !